packages/groq-builder/src/commands/conditional-types.ts

```diff
@@ -25,7 +19,12 @@ export type ConditionalProjectionMap<
       ) => ProjectionMap<TResultItem>);
 };
 
-export type ConditionalExpression<TResultItem> = Tagged<string, TResultItem>;
+/**
+ * For now, none of our "conditions" are strongly-typed,
+ * so we'll just use "string":
+ */
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export type ConditionalExpression<TResultItem> = string;
```

Comment:
This makes sense to me. There might be some value in differentiating between different kinds of strings, but only if there's a strong risk of them getting mixed up with each other.

Reply:
I kinda added this as a placeholder. A conditional expression can be any complex expression,  so there's no way I could enforce it via template strings. But,  I was thinking that I could enable some common auto completions.
For now, this is just a nice way for me to track all the places that were using strings for expressions.